light_recipe/settings.py

```diff
@@ -36,6 +36,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+	
+	'recipes',
```

Comment:
I called django app as cookbook in my PR, so I think we need to resolve this app name convention.